src/libs/actions/Report.js

```diff
@@ -805,6 +805,9 @@ const removeLinks = (comment, links) => {
  */
 const handleUserDeletedLinks = (newCommentText, originalHtml) => {
     const parser = new ExpensiMark();
+    if (newCommentText.length >= 10000) {
```

Comment:
We should move this 10K to CONST

Reply:
I agree

Reply:
There are other places where the number 10000 is being used. I think we should replace all of by them by a const.
https://github.com/Expensify/App/blob/9d04b46456253955c96981db9ef38245953baf18/src/libs/ReportUtils.js#L786